### DISCUSSION TITLE
Website is a little obese

### DISCUSSION THREAD

**Discussion by htor** [zed-industries/zed#18270] [Feature Requests]
Sorry in advance for posting this here, but I couldn't find any links or website repo to create such an issue in.

When I loaded https://zed.dev/ I couldn't help but notice the never-ending spinner around the favicon:
<img width="223" alt="Screenshot 2024-09-24 at 13 35 01" src="https://github.com/user-attachments/assets/bd66586f-37af-4066-af2b-61d427afac33">

Upon further investigation I see the page weighs about 18MB on page load. The culprit is the `default.mp4` which gets loaded in chunks:

<img width="1552" alt="Screenshot 2024-09-24 at 00 44 04" src="https://github.com/user-attachments/assets/6a2d629f-9696-4620-bd07-6bb6982c5c90">

18MB is a bit large. By one order of magnitude. Also, the video should load when I press play, but preferrably not before that.
Other than that, great stuff!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
